backend/migrations/20230703125429_canteen.up.sql

```diff
@@ -1,5 +1,6 @@
 -- Add up migration script here
 CREATE TABLE canteen (
   canteen_id uuid DEFAULT gen_random_uuid() PRIMARY KEY,
-  name text NOT NULL
+  name text NOT NULL,
+  position integer NOT NULL
```

Comment:
It seems practical to define a default position.

Reply:
which position would you choose for that? 0 or a very large number?

Reply:
if we want position to be unique, we cannot /should not do that. even if we dont want it to be enforced to be unique, i dont think a default position makes sense.